@xen-orchestra/lite/src/App.vue

```diff
@@ -126,4 +126,9 @@ const isSslModalOpen = computed(() => unreachableHostsUrls.value.length > 0);
   height: calc(100vh - 9rem);
   background-color: var(--background-color-secondary);
 }
+
+body > .app-menu {
+  background-color: var(--color-blue-scale-500);
+  padding: 0.5rem;
+}
```

Comment:
I think this modification comes from a misusage of the `AppMenu` and can be removed (see my proposal for VmHeader below).

I'll quickly write a documentation for `AppMenu`.